### ISSUE TITLE
Kubernetes complete demo deployment does not include namespace creation

### ISSUE THREAD

**Issue by embs** [microservices-demo/microservices-demo#733]
When I run

    $ kubectl apply -f complete-demo.yaml

as per documentation instructions for running Sock Shop on any Kubernetes cluster, all I've got is a bunch of errors like this one:

    Error from server (NotFound): error when creating "complete-demo.yaml": namespaces "sock-shop" not found

This would probably be solved through something like

    $ kubectl apply -f manifests/sock-shop-ns.yaml -f complete-demo.yaml

But I'd rather expect the complete demo to contemplate the creation of the namespace.

**Comment by RaphaelHerd** [comment]
Yes of course I can add it manually but my understanding of "complete" is complete without editing some lines otherwise it's not usable. I used your demo several times a day to validate our cluster and this is simply not practical. Yes I can fork your repo but my intension is to give you feedback.

So accept the request and it should work in any case......

**Comment by etiennetremel** [comment]
https://github.com/microservices-demo/microservices-demo/issues/746
